openfecwebapp/templates/partials/navigation.html

```diff
@@ -22,6 +22,11 @@
           <span class="site-nav__link__title">Legal resources</span>
         </a>
       </li>
+      <li class="site-nav__item" data-submenu="about">
```

Comment:
Add `site-nav__item--secondary` to prevent this:
<img width="1438" alt="screen shot 2017-07-24 at 6 01 55 pm" src="https://user-images.githubusercontent.com/24054/28546974-b8af25ba-709a-11e7-8444-fa1732b1d5fa.png">